posts/how-to-install-memgraph-and-memgraph-lab-on-windows-10.md

```diff
@@ -0,0 +1,169 @@
+## Step 2 — Installing Memgraph
+ 
+To get started, go to Memgraph's [downolad](https://memgraph.com/download/) page and download the latest Debian installation package. 
+ 
+Once the download is complete, open your Debian shell and run the following command to start the installation process:
+```
+dpkg -i /path/to/memgraph_<version>.deb
+```
+* replace `/path/to` with path where you downloaded your installation package.
+* replace `_version` with the version of the package that you are installing(usually the name of the installation package you downloaded). 
+ 
+For example, if you downloaded version `0.50.0-1_amd64` to your D: Directory than this command will be changed to:
+```
+dpkg -i /mnt/d/memgraph_0.50.0-1_amd64.deb
+```
+Note: If you see any error related to missing dependency packages. You might have to run `sudo apt get update` before installing Memgraph.
+ 
+<p id="run-memgraph-server">Now, start the Memgraph server by issuing the following command:</p>
+ 
+```
+/usr/lib/memgraph/memgraph
+```
+If Memgraph has been installed correctly you will see something like this:
+> Starting 4 BoltS workers
+> BoltS server is fully armed and operational
+> BoltS listening on 0.0.0.0:7687
+ 
+Awesome! You now have a running instance of Memgraph on your Windows machine. In the next step, you will learn how to query Memgraph from the command line interface (CLI).
+ 
+## Step 3 — Querying Memgraph through the CLI tool with mgconsol.
+ 
+Memgraph comes with a tool called `mgconsole` which allows you to execute queries against the Memgraph server from your command-line interface.
+ 
+To install `mgconsole`, you can refer to the [mgconcole installation guide](https://github.com/memgraph/mgconsole)
+ 
+Once you have `mgconsole` successfully installed, you are now ready to query Memgraph. To make sure that everything works, we will execute two basic queries that demonstrate the creation and retrieval of data from the database.
+ 
+Memgraph supports the Cypher query through the  [openCypher](https://www.opencypher.org/) project. Cypher is a SQL-like declarative query language purposely designed to efficiently query and update property graphs.
+ 
+ 
+<p id="first-query">Let's begin by executing a first query to create a small dataset:</p>
+ 
+```
+CREATE (u:User {name: "Alice"})-[:Likes]->(m:Software {name: "Memgraph"});
+```
+<p id="query-explain">The above will create 2 nodes in the database, one labeled "User" with the name "Alice" and the other labeled "Software" with the name "Memgraph". It will also create a relationship that "Alice" likes "Memgraph".</p>
+ 
+Note: Each query needs to end with the `;` (_semicolon_) character.
+ 
+Next, we will retrieve the created nodes and relationships by executing this second query:
+ 
+```
+MATCH (u:User)-[r]->(x) RETURN u, r, x;
+```
+ 
+The query should return the following result:
+```
++--------------------------------+--------------------------------+--------------------------------+
```

Comment:
Please take a look at how this looks here https://github.com/memgraph/blog/blob/memgraph-windows-tutorial/posts/how-to-install-memgraph-and-memgraph-lab-on-windows-10.md. The formatting is broken. Could you please improve the overall layout?